R/abtestbayesian.R

```diff
@@ -68,15 +68,15 @@ ABTestBayesianInternal <- function(jaspResults, dataset = NULL, options) {
   #   The (numeric) columns given as dependent/covariates/wlsWeights
 
   if (!is.null(dataset))
-    return(dataset)
+    return (na.omit(dataset))
 
   asNum             <- c(options$n1, options$y1, options$n2, options$y2)
   dataset           <- .readDataSetToEnd(columns.as.numeric = asNum, excluse.na.listwise = asNum)
```

Comment:
shouldn't `excluse.na.listwise` be `exclude.na.listwise`? That automatically remove the NAs.

Reply:
ah, never mind. I missed it.